apps/tenant-management-webapp/src/app/store/status/api.ts

```diff
@@ -22,13 +22,12 @@ export class StatusApi {
   }
 
   async saveApplication(props: ApplicationStatus): Promise<ApplicationStatus> {
-    if (props.appKey) {
-      const res = await this.http.put(`/applications/${encodeURIComponent(props.appKey)}`, props);
-      return res.data;
-    } else {
-      const res = await this.http.post(`/applications`, props);
-      return res.data;
-    }
+    const res = await this.http.post(`/applications`, props);
+    return res.data;
+  }
+  async updateApplication(props: ApplicationStatus): Promise<ApplicationStatus> {
+    const res = await this.http.put(`/applications/${props.appKey}`, props);
```

Comment:
I would suggest adding the encodeURIComponent to the props.appKey. I might still have old entries there, which might still cause issues.